src/main/java/com/simibubi/create/content/logistics/block/display/source/StatTrackingDisplaySource.java

```diff
@@ -33,8 +40,11 @@ protected Stream<IntAttached<MutableComponent>> provideEntries(DisplayLinkContex
 			scoreboard.addObjective(name, ObjectiveCriteria.DUMMY, getObjectiveDisplayName(), RenderType.INTEGER);
 		Objective objective = scoreboard.getObjective(name);
 
-		sLevel.players()
-			.forEach(s -> scoreboard.getOrCreatePlayerScore(s.getScoreboardName(), objective)
+		Iterator<ServerLevel> serverLevelIterator = ServerLifecycleHooks.getCurrentServer().getAllLevels().iterator();
+		Spliterator<ServerLevel> spliterator = Spliterators.spliteratorUnknownSize(serverLevelIterator, 0);
+		StreamSupport.stream(spliterator, false)
+				.flatMap(l -> l.players().stream())
+				.forEach(s -> scoreboard.getOrCreatePlayerScore(s.getScoreboardName(), objective)
 				.setScore(updatedScoreOf(s)));
```

Comment:
```suggestion
		ServerLifecycleHooks.getCurrentServer().getAllLevels()
				.forEach(l -> l.players()
						.forEach(p -> scoreboard.getOrCreatePlayerScore(p.getScoreboardName(), objective)
						    .setScore(updatedScoreOf(p))));
```
This might be simpler

Reply:
Thanks ^^